### DISCUSSION TITLE
data quality

### DISCUSSION THREAD

**Discussion by jlp2duke** [AgBioData/DataFederation_WG#10] [General]
For curation needs to use evidence codes so that the researcher can make a decision about how confident their are with the data. Our job in this working group is to ‘grease the wheels’ of other working groups - enable the technology for users/databases to be able to access and understand data quality
Example: http://www.candidagenome.org/cgi-bin/GO/goEvidence.pl

**Comment by elserj** [comment]
More details on GO evidence codes:
https://wiki.geneontology.org/index.php/Guide_to_GO_Evidence_Codes
Decision tree for codes:
http://wiki.geneontology.org/index.php/Guide_to_GO_Evidence_Codes#Evidence_codes_Decision_tree

**Comment by jlp2duke** [comment]
From Peter Selby: uses OpenAPI3 as standard for API documentation (https://swagger.io/specification/)

**Reply by adf-ncgr** [reply]
Is this relevant to data quality or was this comment meant for some other context? I see the same comment was made against #9 where it seems more appropriate. If @BrapiCoordinatorSelby agrees perhaps we can remove this subthread from this discussion...

**Reply by jlp2duke** [reply]
I say move to #9

**Reply by BrapiCoordinatorSelby** [reply]
agreed. OpenAPI3 has nothing to do with data quality, it is more about the quality of standards documentation.
